test/Config.js

```diff
@@ -185,7 +252,7 @@ test('validate with values', t => {
           .loader('babel-loader')
           .options({ presets: ['alpha'] });
 
-  const errors = validate(config.toConfig());
-
+  const obj = config.toConfig();
```

Comment:
that line is there for testing with the latest webpack since it errors if it is the latest webpack without absolute paths.